EmployeeManagement/Tavisca.EmployeeManagementService.Implementaion/EMSGetImplementation.cs

```diff
@@ -0,0 +1,31 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Tavisca.EmployeeManagement.DataContract;
+using Tavisca.EmployeeManagement.ServiceContract;
+using Tavisca.EmployeeManagement.Translator;
+using Tavisca.WCF.Model;
+
+namespace Tavisca.EmployeeManagementService.Implementaion
+{
+    class EMSGetImplementation:IEmployeeService
+    {
+        public Employee Get(string id)
+        {
+            EmployeeModel employeeModel = EmployeeModel.GetEmployeeById(id);
```

Comment:
Create a employee manager which will expose the Get and GetAll methods using an interface. Model class should not expose these methods.